kibbeh/src/ui/Search/SearchBar.tsx

```diff
@@ -5,20 +5,26 @@ import { Input } from "../Input";
 export interface SearchBarProps
   extends React.ComponentPropsWithoutRef<"input"> {
   inputClassName?: string;
+  mobile?: boolean;
 }
 
 export const SearchBar: React.FC<SearchBarProps> = ({
   className = "",
   inputClassName = "",
+  mobile = false,
   ...props
 }) => {
   return (
     <div
-      className={`w-full bg-primary-700 text-primary-300 focus-within:text-primary-100 rounded-lg ${className}`}
+      className={`w-full bg-primary-700 text-primary-300 focus-within:text-primary-100 rounded-lg ${
+        mobile ? "px-4" : ""
+      } ${className}`}
     >
-      <div className="h-full mx-4 flex items-center pointer-events-none">
-        <SmSolidSearch />
-      </div>
+      {!mobile && (
+        <div className="h-full mx-4 flex items-center pointer-events-none">
+          <SmSolidSearch />
+        </div>
+      )}
```

Comment:
Similar to above, this can be 

```suggestion
        <div className="h-full mx-4 flex items-center pointer-events-none hidden md:block">
          <SmSolidSearch />
        </div>
```